libs/core/calendar/calendar-views/calendar-month-view/calendar-month-view.component.ts

```diff
@@ -54,15 +56,30 @@ export class CalendarMonthViewComponent<D> implements OnInit, OnChanges, Focusab
     @Input()
     year: number;
 
+    /** Whether to pick the date range in month year format without picking days */
+    @Input()
+    isDateRangeMonthYearFormat: boolean;
+
     /** An event fired when a new month is selected */
     @Output()
     readonly monthClicked: EventEmitter<number> = new EventEmitter<number>();
 
+    /** Event thrown every time selected first or last date in range mode is changed */
+    @Output()
+    readonly selectedRangeDateChange: EventEmitter<DateRange<D>> = new EventEmitter<DateRange<D>>();
+
+    /**
+     * Whether user wants to mark day cells on hover.
```

Comment:
according this feature, user want to select mont or year not a day.

Reply:
@dpavlenishvili to notify the calendar component for a month change or newly month selected